packages/airnode-node/src/api/index.ts

```diff
@@ -287,11 +288,33 @@ export async function callApi(payload: ApiCallPayload): Promise<LogsData<ApiCall
   if (verificationResult) return verificationResult;
 
   const processedPayload = await preProcessApiSpecifications(payload);
-
-  const [logs, response] = await performApiCall(processedPayload);
-  if (isPerformApiCallFailure(response)) {
-    return [logs, response];
+  const ois = payload.config.ois.find((o: OIS) => o.title === payload.aggregatedApiCall.oisTitle) as OIS;
+  const endpoint = ois.endpoints.find((e: Endpoint) => e.name === payload.aggregatedApiCall.endpointName) as Endpoint;
+
+  if (!endpoint.operation && endpoint.fixedOperationParameters.length === 0) {
+    if (
+      (!endpoint.preProcessingSpecifications && !endpoint.postProcessingSpecifications) ||
+      (endpoint.preProcessingSpecifications?.length === 0 && endpoint.postProcessingSpecifications?.length === 0)
+    ) {
+      return [
+        [
+          logger.pend(
+            'ERROR',
+            `Failed to skip API call. Make sure one, or both 'preProcessingSpecifications', 'postProcessingSpecifications' exists and not empty array.`
+          ),
+        ],
+        {
+          success: false,
+          errorMessage: `Failed to skip API call. Make sure one, or both 'preProcessingSpecifications', 'postProcessingSpecifications' exists and not empty array.`,
+        } as SkipApiCallError,
```

Comment:
What's the reason for casting to this specific interface? The function signature specifies that this returns `Promise<LogsData<ApiCallResponse>>` so it's kinda confusing.